sdk/communication/Azure.Communication.NetworkTraversal/tests/SessionRecords/CommunicationRelayClientLiveTests/GettingTurnCredentialsWithConnectionString.json

```diff
@@ -0,0 +1,91 @@
+        "x-ms-content-sha256": "Sanitized",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": {},
+      "StatusCode": 201,
+      "ResponseHeaders": {
+        "api-supported-versions": "2020-07-20-preview2, 2021-02-22-preview1, 2021-03-07, 2021-03-31-preview1",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Mon, 17 May 2021 02:56:40 GMT",
+        "MS-CV": "9SXFadkR9kimX4BHmhpurw.0",
+        "Request-Context": "appId=",
+        "Strict-Transport-Security": "max-age=2592000",
+        "Transfer-Encoding": "chunked",
+        "X-Azure-Ref": "0aNuhYAAAAACVU98zg241T7neY65WSoIAV1NURURHRTA4MDcAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+        "X-Cache": "CONFIG_NOCACHE",
+        "x-ms-client-request-id": "6871f17a0b73943f24d7bf1130628130",
+        "X-Processing-Time": "23ms"
+      },
+      "ResponseBody": {
+        "identity": {
+          "id": "8:acs:e1b50ade-1e68-426f-b602-fdda5fda7eb1_0000000a-1822-b56e-6a0b-343a0d002f3f"
+        }
+      }
+    },
+    {
+      "RequestUri": "https://anpearesource1.communication.azure.com/turn/8%3Aacs%3Ae1b50ade-1e68-426f-b602-fdda5fda7eb1_0000000a-1822-b56e-6a0b-343a0d002f3f/:issueCredentials?api-version=2021-02-22-preview1",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Authorization": "Sanitized",
+        "Date": "Mon, 17 May 2021 02:56:40 GMT",
+        "traceparent": "00-999f97f37584c540ae1908c1987c14c7-c465a567c6d7fd49-00",
+        "User-Agent": [
+          "azsdk-net-Communication.NetworkTraversal/1.0.0-alpha.20210516.1",
+          "(.NET 5.0.6; Microsoft Windows 10.0.19042)"
+        ],
+        "x-ms-client-request-id": "770284ac2cd08e315f4ded0eb88f5dcd",
+        "x-ms-content-sha256": "Sanitized",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": null,
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "api-supported-versions": "2021-02-22-preview1",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Mon, 17 May 2021 02:56:40 GMT",
+        "MS-CV": "RYhAlQlpV0\u002BR9HptbxQZ2Q.0",
+        "Request-Context": "appId=",
+        "Strict-Transport-Security": "max-age=2592000",
+        "Transfer-Encoding": "chunked",
+        "X-Azure-Ref": "0aNuhYAAAAABVtEQ26kCuS5MiIUSpxiFyV1NURURHRTA4MDcAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+        "X-Cache": "CONFIG_NOCACHE",
+        "x-ms-client-request-id": "770284ac2cd08e315f4ded0eb88f5dcd",
+        "X-Processing-Time": "60ms"
+      },
+      "ResponseBody": {
+        "expiresOn": "2021-05-17T02:56:40.9067923\u002B00:00",
+        "turnServers": [
+          {
+            "urls": [
+              "turn:worldaz.turn.skype.com:3478"
+            ],
+            "username": "BQAANpfYYkgB10xaXUCyFBhZ/MN\u002BKFhXh2FJD7IJbiUAAAAMARDhtQreHmhCb7YC/dpf2n6xm5UZ/lYEi2xFd5BBnosQx6j8sNU=",
```

Comment:
Does the user name need to be Sanitized as well?

Reply:
No its just the user Identity hashed with some additional non-sensitive info